documentation/modules/creating-validation-rule.adoc

```diff
@@ -120,7 +120,7 @@ spec:
     namespace: {namespace}
 EOF
 ----
-<1> Allowed values are `ovirt` and `vsphere`.
+<1> Allowed values are `ovirt`, `vsphere`, and `openstack`.
 <2> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere or `https://<engine_host>/ovirt-engine/api/` for {rhv-short}.
```

Comment:
How about also adding `https://<identity_service>/v3 for {osp}` here

Reply:
@qiyuann  @ahadas  -- what about the callout in line 70 in the "Create a config map CR" step?  It  currently reads "<1> Specify the provider package name. Allowed values are `io.konveyor.forklift.vmware` for VMware and `io.konveyor.forklift.ovirt` for {rhv-full}. "  Do we need something for OpenStack (I think so)? If yes, what should it be?

Reply:
@bennyz would it work for openstack? I know we didn't add validations for openstack and I'm not sure how generic this mechanism is to work out-of-the-box with openstack, could you please check?
@RichardHoch I wouldn't add it before we verify it actually works

Reply:
@ahadas Won't add it to this PR unless I hear otherwise. Let's keep this moving towards merge.

Reply:
So after checking it cannot work currently because we are missing two rules (name.rego, validation.rego),
should be possible once:
https://github.com/kubev2v/forklift/issues/258
is addressed

Reply:
ack, thanks @bennyz 
so yeah, let's not add openstack for the time being